storybookjs/storybook
# Fix trim vulnerability

`trim@0.0.1` has a known vulnerability with high severity: https://security.snyk.io/vuln/SNYK-JS-TRIM-1017038.

Here is the path to `trim` in dependencies of the __./code__ folder before this commit:

```
└─ @storybook/root@workspace:.
   └─ @storybook/linter-config@npm:2.5.0 [cfb4f] (via npm:^2.5.0 [cfb4f])
      └─ remark-cli@npm:8.0.1 (via npm:^8.0.0)
         └─ remark@npm:12.0.1 (via npm:^12.0.0)
            └─ remark-parse@npm:8.0.3 (via npm:^8.0.0)
               └─ trim@npm:0.0.1 (via npm:0.0.1)
```

Issue: https://github.com/storybookjs/storybook/issues/14603

## What I did

Upgrade `@storybook/linter-config` to the latest version (`3.1.2`) to fix the `trim` vulnerability issue.

Also, the latest version of `@storybook/linter-config` was used in the __./scripts__ folder before this commit.

## How to test
I checked that there is no `trim` in the dependencies in the __./code__ folder after this commit. You can check it by running the following.

```sh
yarn why -R trim
```


If your answer is yes to any of these, please make sure to include it in your PR.
